### ISSUE TITLE
konnectd stops working after running for some time

### ISSUE THREAD

**Issue by dpakach** [owncloud/ocis-konnectd#41]
After running konnectd for some time and having many requests (for running tests) it stops working and we get following errors.
```
...
2020-02-28T14:29:52+05:45 ERR IdentifierIdentityManager: id_token_hint does not match request error="invalid redirect_uri: http://172.17.0.1:9100/" service=konnectd
2020-02-28 14:40:14.204938 I | http: Accept error: accept tcp [::]:9130: accept4: too many open files; retrying in 5ms
2020-02-28T14:40:14+05:45 ERR IdentifierIdentityManager: fetch failed to get user from userID error="ldap identifier backend get user connect error: LDAP Result Code 200 \"Network Error\": dial tcp: lookup localhost: device or resource busy" service=konnectd
2020-02-28T14:40:14+05:45 ERR identity manager fetch failed error="IdentifierIdentityManager: identifier error" service=konnectd
2020-02-28 14:40:14.210076 I | http: Accept error: accept tcp [::]:9130: accept4: too many open files; retrying in 10ms
2020-02-28 14:40:14.220201 I | http: Accept error: accept tcp [::]:9130: accept4: too many open files; retrying in 20ms
2020-02-28 14:40:14.240877 I | http: Accept error: accept tcp [::]:9130: accept4: too many open files; retrying in 40ms
2020-02-28 14:40:14.281764 I | http: Accept error: accept tcp [::]:9130: accept4: too many open files; retrying in 80ms
2020-02-28 14:40:14.364918 I | http: Accept error: accept tcp [::]:9130: accept4: too many open files; retrying in 160ms
2020-02-28 14:40:14.529093 I | http: Accept error: accept tcp [::]:9130: accept4: too many open files; retrying in 320ms
2020-02-28 14:40:14.849212 I | http: Accept error: accept tcp [::]:9130: accept4: too many open files; retrying in 640ms
2020-02-28 14:40:15.489337 I | http: Accept error: accept tcp [::]:9130: accept4: too many open files; retrying in 1s
2020-02-28 14:40:16.489948 I | http: Accept error: accept tcp [::]:9130: accept4: too many open files; retrying in 1s
2020-02-28 14:40:17.495527 I | http: Accept error: accept tcp [::]:9130: accept4: too many open files; retrying in 1s
2020-02-28 14:40:18.496883 I | http: Accept error: accept tcp [::]:9130: accept4: too many open files; retrying in 1s
2020-02-28 14:40:19.497102 I | http: Accept error: accept tcp [::]:9130: accept4: too many open files; retrying in 1s
2020-02-28 14:40:20.504891 I | http: Accept error: accept tcp [::]:9130: accept4: too many open files; retrying in 1s

```

**Comment by DeepDiver1975** [comment]
Looks Like some sockets are not closed.

**Comment by IljaN** [comment]
We also observed this issue with ocis-proxy, will provide debug-commands and move issue to ocis.

**Comment by IljaN** [comment]
Related https://github.com/owncloud/ocis/issues/268

**Comment by IljaN** [comment]
Fixed with owncloud/ocis#268
